dnf5-plugins/automatic_plugin/bin/dnf-automatic

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/sh
+# Compatibility wrapper to handle old "/usr/bin/dnf-automatic" name from dnf4.
+
+dnf5 automatic "$@"
```

Comment:
What about to mark it as deprecated from beginning?

There is another question whether we should provide it, because it was used by services that were also provided by original package.

Reply:
I'm not completely sure we need it either. The services were also adjusted to the new name.